README.md

```diff
@@ -20,12 +20,13 @@ Before you jump into the code you'll want to download, install, and configure th
 git clone https://github.com/mozilla/webmaker-android
 cd webmaker-android
 npm install
+npm run build
 ```
 
 #### Android
-While the majority of `webmaker-android` is built using Web technologies, it runs within a native Android wrapper that is included as part of this codebase. If you would like to make changes to the wrapper or if you'd like to test the app, we recommend you use [Android Studio](http://developer.android.com/sdk/index.html).
+While the majority of `webmaker-android` is built using Web technologies, it runs within a native Android wrapper which comprises this codebase. To make changes or to test the app, we recommend you use [Android Studio](http://developer.android.com/sdk/index.html).
```

Comment:
this wording is a little funny and assumes knowledge of what a "wrapper" is and maybe elides what a hybrid native app really means. maybe something like "This repository is home to the native Android wrapper for the `webmaker-android` project. `webmaker-android` is a hybrid mobile application that is primarily web-based (HTML/CSS/JS) but uses this wrapper to communicate with the native Android SDK."